src/dictionaries.cpp

```diff
@@ -32,3 +34,17 @@ void DependentDictionary::add(uint8_t type, const std::string& name, const std::
     m_rightDict.emplace(std::make_pair(attributeName, type), name);
     m_reverseDict.emplace(std::make_pair(attributeName, name), type);
 }
+
+Dictionaries::Dictionaries(const std::string& filePath)
+{
+    std::ifstream inf(filePath);
+    if (!inf)
+        throw std::runtime_error("File Dictonary " + filePath + " failed to open.");
```

Comment:
`"Cannot open dictionary file " + filePath`

or

`"Failed to open dictionary file" + filePath`